aggrivate/aggrivate.js

```diff
@@ -29,6 +29,7 @@ var timestarted = Math.floor(new Date() / 1000);
 var stillWaitingForDB = true;
 var waitingForDBCount = 12;
 var feedWorkCount = 0;
+var defaultConfigFilePath = './global.conf';
```

Comment:
Added a default path as a fallback for convenience, here I'm assuming that the CWD is going to contain the global config file. Not sure if this assumption is useful.